lib/jobs.py

```diff
@@ -53,3 +54,49 @@ async def add_job(job_dict):
 
     with litani.atomic_write(jobs_dir / ("%s.json" % job_id)) as handle:
         print(json.dumps(job_dict, indent=2), file=handle)
+
+
+def get_jobs():
+    out = []
+    jobs_dir = litani.get_cache_dir() / litani.JOBS_DIR
+    for job in jobs_dir.iterdir():
+        with open(job) as handle:
+            job_dict = json.load(handle)
+            for key in ("job_id", "status_file", "subcommand"):
```

Comment:
Suggest:
* This tuple is a global variable in this module (but private)
* You reference it here through the variable name
* In `add-job`, just before you write the file, raise `AssertionError` if any of those keys in the tuple are not part of the dict that you're about to write, as that suggests a programming error.

This guards against future mistakes when changing this code.